owncloudApp/src/main/java/com/owncloud/android/MainApp.kt

```diff
@@ -179,7 +182,15 @@ class MainApp : Application() {
         }
     }
 
+    override fun attachBaseContext(base: Context?) {
+        super.attachBaseContext(base)
+        MultiDex.install(this)
+    }
+
     fun startLogIfDeveloper() {
+        if (BuildConfig.DEBUG)
+            Timber.plant(DebugTree(Log_OC::class.java))
+
```

Comment:
We can include this inside next if(isDeveloper),  can't we?

Reply:
Hmm, it makes no real diff. 
When `isDeveloper` you see entries in "Logcat" and "Filelogging" too. 
With "DEBUG" I guess you have no duplication.
...but I'm not sure